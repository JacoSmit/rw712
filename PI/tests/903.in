3
15
5 3 10 5 8 10 3 10 2 5 4 5 3 4 10
15
12 7 29 12 27 29 7 29 4 13 10 12 7 10 29
27
12 7 29 12 27 29 7 29 4 13 10 12 7 10 29 27 14 12 12 10 7 7 4 4 4 4 3
