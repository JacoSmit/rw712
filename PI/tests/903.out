1:
1 ~ 2558
2 ~ 1514
3 ~ 5342
4 ~ 2558
5 ~ 4166
6 ~ 5342
7 ~ 1514
8 ~ 5342
9 ~ 1002
10 ~ 2558
11 ~ 2134
12 ~ 2558
13 ~ 1514
14 ~ 2134
15 ~ 5342
2:
1 ~ 2142
2 ~ 1198
3 ~ 5450
4 ~ 2142
5 ~ 5038
6 ~ 5450
7 ~ 1198
8 ~ 5450
9 ~ 678
10 ~ 2406
11 ~ 1814
12 ~ 2142
13 ~ 1198
14 ~ 1814
15 ~ 5450
3:
1 ~ 7807454
2 ~ 4531270
3 ~ 19582790
4 ~ 7807454
5 ~ 18108574
6 ~ 19582790
7 ~ 4531270
8 ~ 19582790
9 ~ 2580350
10 ~ 8468254
11 ~ 6491086
12 ~ 7807454
13 ~ 4531270
14 ~ 6491086
15 ~ 19582790
16 ~ 18108574
17 ~ 9125422
18 ~ 7807454
19 ~ 7807454
20 ~ 6491086
21 ~ 4531270
22 ~ 4531270
23 ~ 2580350
24 ~ 2580350
25 ~ 2580350
26 ~ 2580350
27 ~ 1942198
